recipes/Kili_fine_tuning_tutorial.ipynb

```diff
@@ -0,0 +1,2081 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    inference_cat_array = ["UNABLE_TO_CLASSIFY"] * 800
</pre><p>where does the 800 come from?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/kili-technology/kili-python-sdk/pull/1259/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='29'/>

Reply:
<p>I'm creating a list with fixed length and want to be able to control where the results will get appended based on a dataset that's 800 records long. So that I don't end up with a mismatch. I'll add n explanation that 800 is based on the dataset length</p>